examples/js/loaders/GLTFLoader.js

```diff
@@ -845,6 +845,24 @@ THREE.GLTFLoader = ( function () {
 
 				}
 
+				if ( source.color ) {
+
+					target.color = source.color.clone();
+
+				}
+
+				if ( source.specular ) {
+
+					target.specular = source.specular.clone();
+
+				}
+
+				if ( source.emissive ) {
+
+					target.emissive = source.emissive.clone();
+
+				}
+
```

Comment:
No strong opinion but how about this? Less LOC addition.

```javascript
for ( var i = 0, il = params.length; i < il; i ++ ) {

    var value = source[ params[ i ] ];

  	target[ params[ i ] ] = value.clone ? value.clone() : value;

}
```

Reply:
What if the target value is a texture?

Reply:
Oops, I was missing texture, thanks. So it'll be like this?

```javascript
for ( var i = 0, il = params.length; i < il; i ++ ) {

    var value = source[ params[ i ] ];
    target[ params[ i ] ] = value.isColor ? value.clone() : value;

}
```

Reply:
Looks perfect!

Reply:
You seem being missing this line

```javascript
var value = source[ params[ i ] ];
```

Reply:
Oops, my mistake

Reply:
```js
      target[  params[ i ]  ] = ( value && value.isColor ) ? value.clone() : value;
```
I think it should like this, if current texture is null

Reply:
Good catch! Would you like to make PR fixing that?

Reply:
Ok, I will make a PR.